<?xml version="1.0"?>
<launch>
	<!-- LOAD PARAMETERS -->
	<rosparam command="load" ns="oberstufenlabor_mechatronik_robot" file="$(find oberstufenlabor_mechatronik_robot)/cfg/params.yaml" />

        <!-- DEFINE POSES.YAML LOCATION -->
        <param name="oberstufenlabor_mechatronik_robot/poses_file" value="$(find oberstufenlabor_mechatronik_robot)/poses/poses.yaml" />

        <!-- LAUNCH CALIBRATION NODE -->
        <node name="calibration_node" pkg="oberstufenlabor_mechatronik_robot" type="collector_calibration_node" output="screen"/>

</launch>
